src/app/header/components/TermButtons.tsx

```diff
@@ -4,20 +4,20 @@ import React, { useContext, useEffect, useState } from 'react';
 import { TermContext } from '../../context/TermContext';
 import { getCurrentTerms, getReadableTerm } from '../../shared/utils/terms';
 
+const terms = getCurrentTerms();
+
 export function TermButtons(): JSX.Element {
   const [status, setStatus] = useState([false, false, false]);
 
   const { term, setTerm } = useContext(TermContext);
 
-  const terms = getCurrentTerms();
```

Comment:
Should this be a React hook (or wrapped into a React hook)?

Reply:
Not sure if it really needs to be. To me this is just a global variable being used for reference in the component.

Reply:
If you're 100% sure we won't have issues where this being used in different places (and not being in sync) will cause any issues then I think we're fine to leave it as is.